app/assets/stylesheets/bootstrap_overrides.less

```diff
@@ -33,16 +33,102 @@ body {
   height: auto !important;
   height: 100%;
   /* Negative indent footer by it's height */
-  margin: 0 auto -@baseFontSize*1.5;
+  margin: 0 auto -@font-size-base*1.5;
 }
 
+.page-header{
+  margin: 20px 0 30px;
+}
+
+.form-report{ 
+  .input-sm {
+    width: 220px;
+   }
+
+   label{
+    font-weight: normal;
+   } 
+}
```

Comment:
form-report provides the width to the input fields in the report view we can remove the page-header as it is only providing the margin